container.fc

```diff
@@ -70,6 +70,7 @@
 /var/lib/containerd(/.*)?	gen_context(system_u:object_r:container_var_lib_t,s0)
 /var/lib/containerd/[^/]*/snapshots(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 /var/lib/containerd/[^/]*/sandboxes(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
+/var/lib/nerdctl(/.*)? gen_context(system_u:object_r:container_ro_file_t,s0)
```

Comment:
How does this work with writable volumes?

Reply:
I add  /var/lib/nerdctl/[^/]*/volumes/[^/]*/.* gen_context(system_u:object_r:container_file_t,s0)

Reply:
How did you test this?